netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -358,6 +469,82 @@ _net_ebpf_sock_addr_update_store_entries()
     NET_EBPF_EXT_RETURN_NTSTATUS(status);
 }
 
+static NTSTATUS
+_net_ebpf_sock_addr_create_security_descriptor()
+{
+    ACL* dacl = NULL;
+    uint32_t acl_length = 0;
+    PGENERIC_MAPPING mapping = NULL;
+    ACCESS_MASK access_mask = GENERIC_ALL;
+    SECURITY_DESCRIPTOR* admin_security_descriptor = NULL;
+
+    mapping = IoGetFileObjectGenericMapping();
+    RtlMapGenericMask(&access_mask, mapping);
+
+    admin_security_descriptor = (SECURITY_DESCRIPTOR*)ExAllocatePoolUninitialized(
+        NonPagedPoolNx, sizeof(SECURITY_DESCRIPTOR), NET_EBPF_EXTENSION_POOL_TAG);
+    if (admin_security_descriptor == NULL) {
+        return STATUS_NO_MEMORY;
+    }
+
+    NTSTATUS status = RtlCreateSecurityDescriptor(admin_security_descriptor, SECURITY_DESCRIPTOR_REVISION);
+    if (!NT_SUCCESS(status)) {
+        return status;
```

Comment:
Missing tracelog?

As mentioned in my comment for another source file in this change-set, it would be helpful if error handling blocks (this and others in this source file) for 'benign' and frequent errors added a comment, in lieu of logging such errors.

This suggestion is purely from the perspective of giving future maintainers a better idea of the code flow :-)

Reply:
Added traces.